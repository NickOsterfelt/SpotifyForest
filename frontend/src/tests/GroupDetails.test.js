import React from "react";
import { render } from "@testing-library/react";
import GroupDetails from "../components/GroupDetails"

it("renders without crashing", function() {
  render(<GroupDetails userData={{id: 1}}/>);
});

it("matches snapshot", function() {
  const { asFragment } = render(<GroupDetails userData={{id: 1}}/>);
  expect(asFragment()).toMatchSnapshot();
});
