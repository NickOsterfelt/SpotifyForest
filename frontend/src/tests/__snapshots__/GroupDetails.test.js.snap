// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="group-control pt-3"
    >
      <h4 />
    </div>
    <div
      class="main pt-2"
    >
      <table>
        <tbody>
          <tr>
            <td
              class="placeholder-tile"
            >
              <div
                class="item"
              />
            </td>
            <td
              class="placeholder-tile"
            >
              <div
                class="item"
              />
            </td>
            <td
              class="placeholder-tile"
            >
              <div
                class="item"
              />
            </td>
          </tr>
          <tr>
            <td
              class="placeholder-tile"
            >
              <div
                class="item"
              />
            </td>
            <td
              class="user-tile curr-user"
            >
              <div
                class="item"
              >
                <div
                  class="header"
                >
                  <h5
                    href="/users/1"
                  >
                    <u>
                      You
                    </u>
                  </h5>
                </div>
                <div
                  class="image-container"
                >
                  <img
                    alt=""
                    src="https://thumbs.dreamstime.com/b/default-avatar-profile-image-vector-social-media-user-icon-potrait-182347582.jpg"
                  />
                </div>
              </div>
            </td>
            <td
              class="placeholder-tile"
            >
              <div
                class="item"
              />
            </td>
          </tr>
          <tr>
            <td
              class="placeholder-tile"
            >
              <div
                class="item"
              />
            </td>
            <td
              class="placeholder-tile"
            >
              <div
                class="item"
              />
            </td>
            <td
              class="placeholder-tile"
            >
              <div
                class="item"
              />
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</DocumentFragment>
`;
